llama-index-core/llama_index/core/postprocessor/sbert_rerank.py

```diff
@@ -20,6 +20,10 @@ class SentenceTransformerRerank(BaseNodePostprocessor):
         default=False,
         description="Whether to keep the retrieval score in metadata.",
     )
+    trust_remote_code: bool = Field(
+        default=True,
```

Comment:
I don't think this should default to `True`, seems like a CVE waiting to happen :)